code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -172,3 +172,11 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	energy_release = 6000000
 	requirements = list(/datum/gas/hypernoblium = 100, /datum/gas/stimulum = 100)
 	products = list(/obj/item/fuel_pellet/exotic = 1)
+
+/datum/gas_recipe/crystallizer/crystal_foam
+	id = "crystal_foam"
+	name = "Crystal foam grenade"
+	reaction_type =  EXOTHERMIC_REACTION
+	energy_release = 140000
+	requirements = list(/datum/gas/carbon_dioxide = 150, /datum/gas/nitrous_oxide = 100, /datum/gas/bz = 50)
```

Comment:
I'd be fine with lower requirements here, maybe h2o instead of bz? or just lower the required bz a bit since this should be opening tier stuff

Reply:
I'll do 25 H2O